Gemfile

```diff
@@ -1,8 +1,6 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.0'
```

Comment:
Because CircleCI still doesn't like Ruby 2.5.0

Reply:
What about `Ruby 2.5.2`?

Reply:
Ruby version should be specified only here: https://github.com/vjtknn/employ/blob/grAPI/.ruby-version and I see you've specified it there